athena-vertica/README.md

```diff
@@ -0,0 +1,4 @@
+## Vertica Athena Connector
+A custom connector that connects to Vertica database
```

Comment:
I know this is an early feedback CR, just noting that we need to come back and add the readme.

Reply:
thanks, will update this as we go ahead.